applications/zpc/components/zpc_stdin/test/zpc_stdin_test.c

```diff
@@ -334,6 +334,7 @@ void test_handle_nls()
 {
   sl_status_t state;
   uint8_t nls_state = false;
+  uint8_t nls_support = false;
```

Comment:
nitpicking(non-blocking): It's odd to me to me. Would you consider fixing `nls_state` as well? Not in the PR, but considering the minor change.

```suggestion
  uint8_t nls_support = 0;
```

Reply:
addressed